src/gwt/tools/i18n-helpers/create_dev_locale.sh

```diff
@@ -0,0 +1,21 @@
+# Creates a "_dev" locale, which is a copy of the _en locale with PREFIX prepended 
+# onto all strings for easy spotting in the UI
+# 
+# Running this script will result in a "XXX_dev.properties" file for each 
+# "XXX_en.properties" file in this directory and its subdirectories
+
+# Note: run this from /src/gwt/src, otherwise you'll create extra properties files
+# from places that are generated code rather than input code
+
+PREFIX="@"
+
+for src in $(find -name "*en.properties"); do
```

Comment:
This won't work on Mac (at least with the built-in `find` command). Unlike Linux, Mac `find` requires specification of the path. So I believe just stick a period between `find` and `-name` and it'll be good.